src/app/my-profile/components/cloud-credentials-editor/gce-cred-editor/gce-cred-editor.component.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    <div class="form" [formGroup]="gceCredentialsForm">
+
+        <div class="alert alert-danger" role="alert" *ngIf="credentials.errors && credentials.errors.json">
+            <span class="glyphicon glyphicon-exclamation-sign"
+                aria-hidden="true"></span> <span class="sr-only">Error:</span> The provided credentials are not valid JSON.
+        </div>
+        <!-- Credentials -->
+        <div class="form-group is-empty"
+         [ngClass]="{'has-error': '!credentials.valid'}">
+            <label for="id_credentials" class="control-label">GCE Credentials</label>
+            <textarea id="id_credentials" name="credentials" class="form-control input-md" formControlName="credentials" rows="12"></textarea>
+            <span class="help-block"> Copy your GCE credentials json file into this field, or use the upload button above.</span>
+            <span class="material-input"></span>
+        </div>
+    </div>
+</div>
```

Comment:
This html was changed to use angular-material  in the other providers: https://github.com/galaxyproject/cloudlaunch-ui/blob/master/src/app/my-profile/components/cloud-credentials-editor/aws-cred-editor/aws-cred-editor.component.html
Would you mind copying it over from there or I can do it if you're short on time.